### DISCUSSION TITLE
Data for legs of a trip?

### DISCUSSION THREAD

**Discussion by Cfrizz1** [jaybo/deepzoom-blog#36] [General]
Is it possible to extract the data (distance and compass bearing) for all the legs of a route? I use DZ for kayak trip planning and often end up putting together a small chart in excel of each leg of the routes like this: 
![image](https://github.com/jaybo/deepzoom-blog/assets/154295648/3ceaf042-8b14-4e30-b950-ae30590183d2)
Is this possible within Deep Zoom itself?

Thx!!
![Screenshot 2023-12-18 191347](https://github.com/jaybo/deepzoom-blog/assets/154295648/be34f5a2-6642-4f3a-af60-d720479b0b97)

**Comment by jaybo** [comment]
There isn't a way to export this data as a table at present.
But hopefully you've found that you can display the leg length and bearing (magnetic and true) by turning on the corresponding toggles on the settings page.

![image](https://github.com/jaybo/deepzoom-blog/assets/1607323/9fcabb21-a1c0-4e02-9257-b60d4d2a4a36)

**Reply by Cfrizz1** [reply]
Thanks for the reply, I do have that checked but depending on the zoom level when I print my chart  they don't all show or sometimes they are hard to read in a busy chart area.
